src/jit/importer.cpp

```diff
@@ -11226,6 +11226,23 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 #endif // _TARGET_64BIT_
                 }
 
+#ifdef FEATURE_CORECLR
+                if ((oper == GT_ADD) || (oper == GT_SUB) || (oper == GT_MUL) || (oper == GT_DIV) || (oper == GT_MOD))
```

Comment:
> How to determine that operation is unsigned?

I think you may only be missing `GT_UMOD` and `GT_UDIV`. We might not even have those operators at this point, so it may not be necessary in any case.